/**:
  ros__parameters:
    step_time_period_sec: 0.05
    non_critical_log_time_period_ms: 2000
    use_perception: false
    use_params: true
    start_in_pit: true
    num_start_laps: 0
    cte_threshold: 4.0
    cte_merge_threshold: 2.2
    lap_threshold: 100.0
    passing_sf_threshold: 100.0
    back_straight:
      start_percentage: 40.0
      end_percentage: 58.0
    manual_overrides:
      auto_enabled: true
      limit_auto_throttle: true
    timeout:
      manual:
        localization_timeout: 0.5
        low_level_fault_timeout: 0.5
        race_control_timeout: 3.0
        input_manual_command_timeout: 1.0
        opp_car_detections_timeout: 0.5
      autonomous:
        localization_timeout: 0.5
        low_level_fault_timeout: 0.5
        race_control_timeout: 5.0
        input_manual_command_timeout: 3.0
        opp_car_detections_timeout:  0.5
    behavior_tree:
      zmq_max_msgs_per_sec: 100
      groot_publisher_port: 1666
      groot_server_port: 1667
      groot_ip_addr: "*"
    ttls:
      left_ttl_index: 2
      right_ttl_index: 9
      optimal_ttl_index: 18
      race_ttl_index: 2
      pit_ttl_index: 27
    speed_limit:
      pit_lane: 20.0
      pit_road: 40.0
      pit_crawl: 10.0
      yellow: 40.0
      green: 50.0
      max: 200.0
    gaps:
      object_detection: 60.0
      attacker_preparing: 31.0
      attacker_attacking: 13.0
      defender_overtaken: 20.0
      defender_vicinity: 10.0
      attacker_done: 30.0
    speed_modifiers:
      overtaking: [20.0, 15.0]
      speed_threshold_for_overtaking: [ 0.0, 150.0]
      catchup: [20.0, 15.0]
      speed_threshold_for_catchup: [ 0.0, 150.0]
    target_speed_profiles:
      enabled: false
      tsp_name: "lvms"
      tsp_segment_names: ["start", "end"]
      lap_percentages: [0.0, 100.0] 
      speed_profiles: [8.0, 10.0]
    scale_factors:
      enabled: true
      lap_percentages: [0.0, 100.0]
      scale_factors: [100.0, 100.0]
    passing_lines:
      percentages: [72.0]
    passing_zones:
      enabled: true
      start_percentages: [40.0, 82.5]
      end_percentages: [58.0, 13.0]
      max_speeds: [250.0, 160.0]
